lib/controllers/widget/device_list.dart

```diff
@@ -145,20 +193,18 @@ class _DeviceListState extends State<DeviceList> {
       appBar: AppBar(
         title: Text('Chrome OS Devices'),
       ),
-      body: _loading == true && _list == null
-          ?
-          // Loading screen if its the first loading of the list
-          Container(
-              child: Center(
-                child: Text('Loading...'),
-              ),
-            )
-          : Container(
-              child: _errorOnLoading == true
-                  ?
-                  // Warning notification when a error on loading ocurrs
-                  Center(child: Text('$_warning'))
-                  : _list.chromeosdevices.length == 0
+      body: _errorOnLoading
+          ? _warning.statusCode > 499
+              ? _alertAndRetry(_warning)
+              : _alertAndLogIn(_warning, false)
+          : _loading == true && _list == null
```

Comment:
Is it possible for loading to be false and list to be null?

Reply:
No, because it will first call the API to get a value and it will return a value unless it gets an error which is already cached.